src/bot.py

```diff
@@ -107,16 +108,11 @@ async def run_tasks(self, contributor: Contributor) -> None:
         else:
             logging.warning("No contributor for the given time period.")
 
-    def run(self) -> None:
-        """
-        Run the required functions in order for the bot.
-        """
+    @staticmethod
+    @aiocron.crontab("0 0 1 * *")
```

Comment:
I don't see where you are using the environment variable DURATION

Reply:
Well we don't need the environment variable do we if we schedule it at the first of every month?

Reply:
We are not making it to run every month .. It is flexible 
we can set it to every day, week, month, fortnight, year, decade, whatever :D

Reply:
> We are not making it to run every month .. It is flexible we can set it to every day, week, month, fortnight, year, decade, whatever :D

Gotcha captain :v: